torch/csrc/jit/script/module.h

```diff
@@ -381,120 +391,237 @@ struct TORCH_API Module {
   mutable ModulePtr module_value_;
 };
 
-struct NameModule {
-  std::string name;
-  Module module;
-};
+namespace detail {
 
-struct NameValue {
-  std::string name;
-  IValue value;
+struct Frame {
```

Comment:
The name `Frame` is already overloaded in various contexts, are we sure we want to use it here as well?

Reply:
Ill change it to something less ambiguous.